book/src/project-policies.md

```diff
@@ -1,36 +1,69 @@
-# Kubernetes and CSI Sidecar Compatibility
+# Versioning, Support, and Kubernetes Compatibility
 
-Every version of a sidecar has a minimum, maximum and recommended Kubernetes version
+## Versioning
+
+Each Kubernetes CSI component version is expressed as *x.y.z*, where *x* is the
+major version, *y* is the minor version, and *z* is the patch version,
+following [Semantic Versioning](https://semver.org/).
+
+Patch version releases only contain bug fixes that do not break any backwards
+compatibility.
+
+Minor version releases may contain new functionality that do not
+break any backwards compatibility (except for alpha features).
+
+Major version releases may contain new functionality or fixes that may break
+backwards compatibility with previous major releases. Changes that require a
+major version increase include: removing or changing API, flags, or behavior, new
+RBAC requirements that are not opt-in, new Kubernetes minimum version
+requirements.
+
+A litmus test for not breaking compatibility is to replace the image of a
+component in an existing deployment without changing that deployment in any
+other way.
+
+## Support
+
+The Kubernetes CSI project follows the broader Kubernetes project on support.
+Every minor release branch will be supported with patch releases for at least 1 year,
```

Comment:
> will be supported with patch releases

It might be worth clarifying further what that means - and then we can check whether we actually live up to that promise.

In practice, "still supported" means that cherry-picks for bug fixes are accepted and new releases tagged if someone cares to create the PRs and asks for a release.

There is no systematic monitoring of bug fixes in master to determine which of those should be backported and to which older releases.

Reply:
added "on as as-needed" basis